exporter/sumologicexporter/README.md

```diff
@@ -0,0 +1,49 @@
+# Sumo Logic Exporter
+
+This exporter supports sending logs and metrics data to [Sumo Logic](https://www.sumologic.com/).
+
+The following configuration options are supported:
+
+- `url` (required): Unique URL generated for your HTTP Metrics Source. This is the address to send metrics to.
+- `method` (optional): HTTP method, one of: `POST` or `PUT`. `POST` is used by default if unset.
+- `compress` (optional): Option to enable compression (default false)
+- `compress_encoding` (optional): Compression encoding format, either gzip or deflate (default gzip)
+- `max_request_body_size` (optional): Max HTTP request body size in bytes before compression (if applied). By default 1MB is recommended.
+- `log_format` (optional) (logs only): Format to post logs into Sumo. (default `json`)
+  - `text` - Logs will appear in Sumo Logic in text format
+  - `json` - Logs will appear in Sumo Logic in json format.
+- `metric_format` (optional) (metrics only): The format of metrics you will be sending, either graphite or carbon2 or prometheus (Default is graphite).
+- `source_category` (optional): Desired source category. Useful if you want to override the source category configured for the source.
+- `source_name` (optional): Desired source name. Useful if you want to override the source name configured for the source.
+- `source_host` (optional): Desired host name. Useful if you want to override the source host configured for the source.
+- `timeout` (default = 5s): Is the timeout for every attempt to send data to the backend.
```

Comment:
can we check with the collection team and ask what their respective defaults are here and the retry and sending queues? They may have opinions on what works best with Receiver.

Reply:
ALB timeout is 55 sec.